src/pages/_proxied-dot-io/packer/plugins/[...page].tsx

```diff
@@ -25,7 +25,7 @@ const basePath = 'plugins'
 const navDataFile = `data/${basePath}-nav-data.json`
 const localContentDir = `../content/${basePath}`
 const additionalComponents = { Badge, BadgesHeader, PluginBadge, Checklist }
-const mainBranch = 'master'
+const mainBranch = 'stable-website'
```

Comment:
Have switched from `master` to `stable-website` for the dot-io site, as I believe that's more correct.